dvc/analytics.py

```diff
@@ -2,13 +2,18 @@
 
 from __future__ import unicode_literals
 
-from dvc.utils.compat import str
-
 import os
+
```

Comment:
You shouldn't separate individual imports, but rather groups of imports. So this one should be:
```
from __future__ import unicode_literals

import os
import json
import errno
import logging

from dvc.utils.compat import str
from dvc import __version__

```

Reply:
Done accordingly to this.